ftl/core/browsing.ftl

```diff
@@ -149,6 +149,13 @@ browsing-sidebar-save-current-search = Save Current Search
 browsing-sidebar-card-state = Card State
 browsing-sidebar-flags = Flags
 browsing-today = Today
+browsing-tooltip-card-modified = The last time changes were made to a card, including reviews, flags and deck changes
```

Comment:
```suggestion
browsing-tooltip-card-modified = The last time changes were made to a/the card, including reviews, flags and deck changes
```

Reply:
This reads a bit awkwardly - in the notes mode case it should presumably be "a card", but in the cards mode case "this card". Better suggestions welcome!